src/energy_comms/extract/qcew_2010_2021.py

```diff
@@ -0,0 +1,56 @@
+# import dispatch
+import glob
+import io
+import os
+import zipfile
+from io import BytesIO
+
+# import qcew_2014_2021
+# from qcew_2014_2021 import *
+from pathlib import Path
+
+import pandas as pd
+import requests
+
+from energy_comms.constants import YEARS
+
+"""Function that downloads zip file through an online URL"""
+
+# years = ['2010','2011','2012','2013','2014','2015','2016','2017','2018','2019','2020','2021']
+CACHE_DIR = Path.home() / "Documents/qcew_cache"
```

Comment:
Is it necessary to save these files locally or is that just useful for development? I think we could get rid of this cache so a bunch of files aren't saved every time.